EmbeddedDocumentsQuery/src/EmbeddedDocumentsQuery.java

```diff
@@ -1,169 +0,0 @@
-
-public class EmbeddedDocumentsQuery {
-	
-	/**
-	 * Retrieves documents filtered by shipping address - city.
-	 * @param connectionString: To MongoDB instance/MongoDB Cluster.
-	 * @param city: Shipping address - city to filter documents.
-	 * @return List<Document>: ArrayList of matching documents.
-	 */
-	
-	public List<Document> FindOrdersByCity(String connectionString, String city)
-	{
-		if(connectionString == null || connectionString.isEmpty() || city==null|| city.isEmpty())
-			{
-				throw new IllegalArgumentException();
-			}
-		
-		MongoClientURI clientUri = new MongoClientURI(connectionString);
-		try(MongoClient client = new MongoClient(clientUri))
-		{
-			MongoDatabase database = client.getDatabase("stores");			
-			MongoCollection<Document> collection = database.getCollection("orders");		
-			
-			List<Document> queryQrders = collection
-					.find(eq("shippingAddress.city",city))
-					.into(new ArrayList<Document>());
-			
-			return queryQrders;
-		}
-		catch (Exception e) 
-		{
-			//log the exception
-	    	System.out.println("An exception occured");
-	        System.out.println("Details:");
-	        System.out.println(e.getStackTrace());
-			return null;
-		}		
-	}
-
-	
-	/**
-	 * Retrieves documents filtered by shipping address - state and city.
-	 * @param connectionString: To MongoDB instance/MongoDB Cluster.
-	 * @param state: Shipping address - state to filter documents.
-	 * @param city: Shipping address - city to filter documents.
-	 * @return List<Document>: ArrayList of matching documents.
-	 */
-	
-	public List<Document> FindOrdersByStateAndCity(String connectionString, String state, String city)
-	{
-        if(connectionString == null || connectionString.isEmpty() || state.isEmpty() || state ==null || city.isEmpty() || city ==null)
-        	{
-        		throw new IllegalArgumentException();
-        	}
-        
-		MongoClientURI clientUri = new MongoClientURI(connectionString);
-		try(MongoClient client = new MongoClient(clientUri))
-		{
-			MongoDatabase database = client.getDatabase("stores");			
-			MongoCollection<Document> collection = database.getCollection("orders");		
-			
-			List<Document> queryQrders = collection
-					.find(and(eq("shippingAddress.state",state),eq("shippingAddress.city",city)))
-					.into(new ArrayList<Document>());
-			
-			return queryQrders;
-		}
-		catch (Exception e) 
-		{
-			//log the exception
-	    	System.out.println("An exception occured");
-	        System.out.println("Details:");
-	        System.out.println(e.getStackTrace());
-			return null;
-		}		
-	}
-	
-	/**
-	 * Retrieves documents filtered by lineitems - unit_price. Use of gt
-	 * @param connectionString: To MongoDB instance/MongoDB Cluster.
-	 * @param unit_price: lineitems - unit_price to filter documents.
-	 * @return List<Document>: ArrayList of matching documents.
-	 */
-	 
-	public List<Document> FindOrdersByUnitPrice(String connectionString, Double unit_price)
```

Comment:
Use the same naming convention for all the variables.

unit_price => unitPrice